### PR TITLE
refactor: extract cors configs into dify config and cleanup the config class

### PR DESCRIPTION
**Pull request by bowenliang123**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of Change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update, included: [Dify Document](https://github.com/langgenius/dify-docs)
- [ ] Improvement, including but not limited to code refactoring, performance optimization, and UI/UX improvement
- [ ] Dependency upgrade

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] TODO

# Suggested Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes generate no new warnings
- [ ] I ran `dev/reformat`(backend) and `cd web && npx lint-staged`(frontend) to appease the lint gods
- [ ] `optional` I have made corresponding changes to the documentation 
- [ ] `optional` I have added tests that prove my fix is effective or that my feature works
- [ ] `optional` New and existing unit tests pass locally with my changes

### CODE REVIEW

File: api/config.py [hunk 1]
```diff
@@ -1,41 +0,0 @@
...
-    return cors_allow_origins
-
-
-class Config:
-    """Application configuration class."""
-
-    def __init__(self):
-        dotenv.load_dotenv()
-
-        self.TESTING = False
```
Comment (takatost) [inline api/config.py:35]: It seems like the parameter TESTING is missing

File: api/core/helper/code_executor/code_executor.py [hunk 2]
```diff
@@ -18,8 +18,8 @@
 logger = logging.getLogger(__name__)
 
 # Code Executor
-CODE_EXECUTION_ENDPOINT = get_env('CODE_EXECUTION_ENDPOINT')
-CODE_EXECUTION_API_KEY = get_env('CODE_EXECUTION_API_KEY')
+CODE_EXECUTION_ENDPOINT = os.environ.get('CODE_EXECUTION_ENDPOINT', 'http://sandbox:8194')
```
Comment (takatost) [inline api/core/helper/code_executor/code_executor.py:21]: Using `current_app.config.get()` would be better

**Review by takatost — CHANGES_REQUESTED** [review]

File: api/config.py (hunk 1 above)
Comment (bowenliang123) [reply api/config.py:35]: `TESTING` seems a legacy config and has no proper use.

File: api/core/helper/code_executor/code_executor.py (hunk 2 above)
Comment (bowenliang123) [reply api/core/helper/code_executor/code_executor.py:21]: I have tried to use `current_app.config.get()` at first. However, it blocks the unit tests, as no Flask app context at the runtime.

File: api/config.py (hunk 1 above)
Comment (bowenliang123) [reply api/config.py:35]: Moved `TESTING` into DeploymentConfig , as you requested.

**Review by takatost — DISMISSED** [review]
LGTM

**Review by takatost — APPROVED** [review]

**Pull request merged**